device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/InternalClient.java

```diff
@@ -1216,4 +1221,101 @@ private void verifyMethodsAreSupported()
             throw new UnsupportedOperationException(METHODS_OVER_HTTP_ERROR_MESSAGE);
         }
     }
+
+    /**
+     * Sends the TelemetryMessage to IoT hub.
+     * @param telemetryMessage The user supplied telemetry message.
+     */
+    public void sendTelemetry(TelemetryMessage telemetryMessage)
```

Comment:
Is this a PnP specific API? The naming is vague enough that I'd expect customers to be confused and use this to send non-pnp telemetry

Reply:
Do we have an existing send events API that doesn't take in a callback? Without the callback this API behaves as fire-and-forget. I wonder how useful it would be for users.

Reply:
We do have some existing methods that are fire and forget.